lib/ip_wrangler/version.rb

```diff
@@ -0,0 +1,3 @@
+module Redirus
+  VERSION = '0.0.1'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.